content-test/addon/Feeds/HighlightsFeed.test.js

```diff
@@ -107,6 +107,15 @@ describe("HighlightsFeed", () => {
           assert.equal(action.type, "HIGHLIGHTS_AWAITING_METADATA");
         });
     });
+    it("should resolve with a HIGHLIGHTS_RESPONSE action if we've tried more than 5 times", () => {
+      instance.retryHighlightsCount = 6;
```

Comment:
If you set this to 4 then call getData once to show it returns AWAITING_METADATA and then again to show that it switches to HIGHLIGHTS_RESPONSE that'll test the transition more rigorously.